packages/data-point/test/definitions/requests.js

```diff
@@ -40,17 +40,9 @@ module.exports = {
     value: () => ({ source: 'source5' }),
     url: 'http://remote.test/{value.source}'
   },
-  'request:a2': {
```

Comment:
acc.url was an undocumented behaviour that to my opinion can be deprecated without being marked as a breaking change